buildSrc/src/main/kotlin/deps.kt

```diff
@@ -113,6 +114,7 @@ object deps {
         const val roomCompiler = "androidx.room:room-compiler:${versions.room}"
         const val roomTesting = "androidx.room:room-testing:${versions.room}"
         const val roomKtx = "androidx.room:room-ktx:${versions.room}"
+        const val sqlite = "org.xerial:sqlite-jdbc:3.34.0"
```

Comment:
is this needed?

Reply:
this is added to fix `java.lang.Exception: No native library is found for os.name=Mac and os.arch=aarch64. path=/org/sqlite/native/Mac/aarch64` for mac m1